server/src/test/java/io/crate/integrationtests/SumAggregationCastingIT.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.integrationtests;
+
+import static io.crate.testing.Asserts.assertThat;
+
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.Test;
+
+@IntegTestCase.ClusterScope(numDataNodes = 2, supportsDedicatedMasters = false, numClientNodes = 0)
+public class SumAggregationCastingIT extends IntegTestCase {
```

Comment:
Since we have unit tests, maybe just add the test to the existing:
`AggregateExpressionIntegrationTest`?
(Although we don't force at least 2 nodes currently, maybe just add `minDataNodes` to the existing class, so we could catch such issues earlier?

Reply:
👍  and I'd also only limit the setup by whats needed, so probably `minDataNodes` only.